diff --git a/node_modules/@opengsn/contracts/types/truffle-contracts/types.d.ts b/node_modules/@opengsn/contracts/types/truffle-contracts/types.d.ts
index 168521f..ab9d1c1 100644
--- a/node_modules/@opengsn/contracts/types/truffle-contracts/types.d.ts
+++ b/node_modules/@opengsn/contracts/types/truffle-contracts/types.d.ts
@@ -4,8 +4,16 @@
 /**
  * Globals
  */
-/// <reference types="chai" />
-/// <reference types="mocha" />
+// We don't have chai and mocha installed. Shim mocha and remove usage of chai, due to a type collision of assert and
+// expect.
+// /// <reference types="chai" />
+// /// <reference types="mocha" />
+declare namespace Mocha {
+  type Suite = any;
+  type SuiteFunction = any;
+  type ExclusiveSuiteFunction = any;
+  type PendingSuiteFunction = any;
+}
 
 declare type EventEmitter = import("events").EventEmitter;
 
@@ -17,8 +25,8 @@ declare type PromiEvent<T> = import("web3-core").PromiEvent<T>;
 declare type TransactionReceipt = import("web3-core").TransactionReceipt;
 declare type Web3EventOptions = import("web3-eth-contract").EventOptions;
 
-declare const assert: Chai.AssertStatic;
-declare const expect: Chai.ExpectStatic;
+// declare const assert: Chai.AssertStatic;
+// declare const expect: Chai.ExpectStatic;
 
 declare const web3: Web3;
 
